src/quo/design_system/colors.cljs

```diff
@@ -179,8 +179,8 @@
                      purple
                      "#8B3131"])
 
-(def mention-incoming "#0DA4C9")
-(def mention-outgoing "#9EE8FA")
+(def mention-incoming "#4360DF")
+(def mention-outgoing "#4360DF")
```

Comment:
we shouldn't make changes in `quo/...`. Please use colors from `quo2.foundations.colors`